DataStructures/List.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+
+namespace DataStructures
+{
+    public class List<T>
+    {
+        private T[] items;
+        private int count;
+
+        public List()
+        {
+            items = new T[5];
+            count = 0;
+        }
+        public List(int index)
```

Comment:
This constructor should receive the initial quantity, so when you crea a new object `new List<int>(15)` you are saying that you want a List that have initially 15 espaces to fill

Reply:
Yes, but in really that is only to allow insert a number at create a new list.